Doc/tutorial/errors.rst

```diff
@@ -551,3 +551,65 @@ the following pattern::
    ...    raise ExceptionGroup("Test Failures", excs)
    ...
 
+
+Enriching Exceptions with Notes
+===============================
+
+When an exception is created in order to be raised, it is usually initialized
+with information that describes the error that has occurred. There are cases
+where it is useful to add information after the exception was caught. For this
+purpose, exceptions have a mutable field ``__note__`` that can be string or
```

Comment:
The phrasing "mutable field" seems odd to me. Python uses the term attribute. And you assign new values to attributes. The attribute value itself is not mutable.

"exceptions have a ``__note__`` attribute that can be assigned a string or ``None`` value" perhaps?